app/services/author_notifier_service.rb

```diff
@@ -18,6 +18,13 @@ def notify!
         edition_admin_url,
         public_document_url,
       ).deliver_now
+    rescue Notifications::Client::BadRequestError => e
+      if !%w[integration staging].include?(ENV["SENTRY_CURRENT_ENV"]) || e.message !~ /team-only API key/
+        # in production we care about all errors
+        # in staging and integration the team-only error is unrecoverable when running asynchronously
+        # (team-only error is unrecoverable in production too, but almost certainly impossible)
+        raise
+      end
```

Comment:
```suggestion
    rescue Notifications::Client::BadRequestError => e
      # in production we care about all errors
      # in staging and integration the team-only error is unrecoverable when running asynchronously
      # (team-only error is unrecoverable in production too, but almost certainly impossible)
      raise if !%w[integration staging].include?(ENV["SENTRY_CURRENT_ENV"]) || e.message !~ /team-only API key/
```

Reply:
I was in two minds about this as I thought it drew more attention to the comment, but on balance 👍